examples/astradb/load.ts

```diff
@@ -10,9 +10,9 @@ const collectionName = "movie_reviews";
 async function main() {
   try {
     const reader = new PapaCSVReader(false);
-    const docs = await reader.loadData("../data/movie_reviews.csv");
+    const docs = await reader.loadData("./data/movie_reviews.csv");
```

Comment:
astradb folder doesn't have a data folder anymore - please use the one from the parent dir

Reply:
My readme instructions assumed the user was running this from the `examples` directory but can change that if you think that would be better.

Reply:
oops you're right, good catch!